src/cdk-experimental/dialog/dialog-ref.ts

```diff
@@ -62,7 +62,7 @@ export class DialogRef<T, R = any> {
   }
 
   /** Gets an observable that emits when the overlay's backdrop has been clicked. */
-  backdropClick(): Observable<void> {
+  backdropClick(): Observable<MouseEvent> {
```

Comment:
Could this ever be a `TouchEvent`?

Reply:
Nope, AFAIK mobile browsers just fake it as a `MouseEvent`.